### DISCUSSION TITLE
Inaugural Nebari community meeting on 6th Dec 2022! 💎

### DISCUSSION THREAD

**Discussion by pavithraes** [nebari-dev/nebari#1582] [Community meetings]
We're excited to invite you to the first-ever Nebari community meeting! 🎉

This meeting is for everyone interested in Nebari, including users, contributors, aspiring contributors, maintainers, and curious enthusiasts to share projects, ideas, and feedback!

🗓️ We'll meet on **6th December, at 15:30 UTC**, and then every other week at the same time!

💻 You can join the meeting with this link: https://meet.google.com/rio-ngrj-kyh

📝 We'll take notes on HackMD during the meeting: https://hackmd.io/@nebari/rySM3mLPs

You can keep up with upcoming community meetings through the Nebari community calendar: [docs/community#community-meetings](https://www.nebari.dev/docs/community/#community-meetings), or the "🪴 Community meetings" GitHub Discussions category. :)

See you soon! ✨

**Comment by pavithraes** [comment]
@rsignell-usgs Thank you for being a long-time community member! It'll be awesome to see you in this meeting or any of the upcoming ones. :) 🌻

**Comment by dharhas** [comment]
@alimanfoo You may be interested in this as well.

**Comment by pavithraes** [comment]
🔔  Reminder: The first Nebari community meeting is in a few hours!

**Comment by limacarvalho** [comment]
I would love to join, but I unfortunately have a conflicting meeting at that time. I'm looking forward to the next ones 😎 🙏

**Comment by alimanfoo** [comment]
Hi folks, apologies I couldn't join today due to a meeting clash, but hope to join the next one.

**Comment by pavithraes** [comment]
Thanks to everyone who joined! It was nice to meet you all, and we look forward to seeing you in the next meeting soon!

**📝 Some notes from the meeting**

- Announcements/updates:
    - Released [2022.11.1](https://github.com/nebari-dev/nebari/releases/tag/2022.11.1) last week, planning to do a release every month.
    - Roadmap: https://github.com/nebari-dev/governance/blob/main/roadmap.md
    - We’re working on updating [conda-store](https://conda.store/)!
- Discussion:
    - Dharhas presented the new UI for conda-store, Chris shared a live demo of the conda-store UI (standalone react app) - hoping to integrate in an upcoming release
    - Rich - QHub -> Nebari install transition has not been smooth
        - See https://github.com/orgs/nebari-dev/discussions/1587#discussioncomment-4324204
        - Idea :bulb: run user research on the new CLI (guided init) as a few of the options were confusing or there were duplicated requests for information
    - Ahmad - recently deployed nebari again
        - got Helm extensions working - currently trying to integrate Airflow with Keycloack (also runs MLFlow as a Helm extension 👀)
    - Improve the Backup and restore story for Nebari?

Complete notes: [hackmd.io/@nebari/rySM3mLPs](https://hackmd.io/@nebari/rySM3mLPs)
